### PR TITLE
Adds locked icons, fixes several other icons, and a few other QoL tweaks to Suit Storage Units

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Surprisingly, no issue reports on these:
Broken Super_UV icon (big flashing red overlay), no icon to indicate its locked, and unscrewing it straight up broke the whole icon!
That's all fixed, along with some quality of life, including warnings explaining WHY stuff failed...
I even made the super_uv ACTUALLY output black smoke, as the description says it does, because its cool.

<details><summary>Locked</summary>

image
</details><details><summary>Decontamination (Normal)</summary>

https://user-images.githubusercontent.com/76465278/152286461-ec5d7fa2-fafe-4b08-9048-8f8e8a9e92b8.mp4
</details><details><summary>Decontamination (Hacked)</summary>

https://user-images.githubusercontent.com/76465278/152286668-65072dc4-dbb6-493a-b40b-a278231d53b6.mp4

https://user-images.githubusercontent.com/76465278/152286722-889a7955-1497-4962-9cfd-ed53e241abd7.mp4

</details><details><summary>Panels</summary>

image
</details>

## Why It's Good For The Game
Tidies up something that sees more use now that we have MODsuits (and potentially will get them buildable if that other PR goes thru)

## Changelog

:cl:
imageadd: The Suit Storage Units now have a Locked indicator on their lights! In exchange, decontamination above safe levels has a slightly different indicator (flashing red/yellow)
qol: SSU gives chat warnings as to why it blocks decontamination (no items/safeties on)
fix: SSU panel/super-decontamination icons work properly now
add: The SSU super-decontamination also creates a plume of black smoke at the end, as the flavor description said it did.
/:cl:

### CODE REVIEW

**Comment by OrionTheFox** [comment]
> 

stfu this is fix/qol >:(
plus god i needed a break from hatstacker it hates me and i've grown to hate it

File: code/game/machinery/suit_storage_unit.dm [hunk 1]
```diff
@@ -385,6 +390,9 @@
 		if(uv_super)
 			visible_message(span_warning("[src]'s door creaks open with a loud whining noise. A cloud of foul black smoke escapes from its chamber."))
 			playsound(src, 'sound/machines/airlock_alien_prying.ogg', 50, TRUE)
+			var/datum/effect_system/smoke_spread/bad/black/smoke = new
```
Comment (LemonInTheDark) [inline code/game/machinery/suit_storage_unit.dm:393]: you uh, you sure you want the effects of this?

File: code/game/machinery/suit_storage_unit.dm [hunk 2]
```diff
@@ -278,8 +280,10 @@
 				close_machine()
 		if ("disinfect")
 			if (occupant && safeties)
+				say("Alert: safeties triggered, occupant detected!")
```
Comment (LemonInTheDark) [inline code/game/machinery/suit_storage_unit.dm:283]: does this show up in runechat?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Comment by LemonInTheDark** [comment]
Also, document the smoke in your changelog pls

File: code/game/machinery/suit_storage_unit.dm (hunk 1 above)
Comment (OrionTheFox) [reply code/game/machinery/suit_storage_unit.dm:393]: /bad/ just means that it causes coughing, which seems fitting to opening something that just absolutely melted stuff that you PROBABLY shouldnt be inhaling. Plus its only on the same tile as the machine itself. So I dont see a reason not to;

plus this adds a bad/black subtype for future use

File: code/game/machinery/suit_storage_unit.dm (hunk 2 above)
Comment (OrionTheFox) [reply code/game/machinery/suit_storage_unit.dm:283]: Yup, the joys of Say; it does a to-chat AND a runechat message

File: code/game/machinery/suit_storage_unit.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/game/machinery/suit_storage_unit.dm:393]: Doesn't it make you drop your items? That's most of what I was worried about. being on one tile is less concerning

Comment (OrionTheFox) [reply code/game/machinery/suit_storage_unit.dm:393]: Yeaaaa it does, just saw it. But again, being on one tile and doing that more just means if anyone goes to take a DEEP whiff of that molten plasteel then they'll cough-drop it, which again makes sense.
If you want, I can move it back to a normal black-colored smoke though

**Pull request merged**